### ISSUE TITLE
Make some CodeLabs to improve onboarding

### ISSUE THREAD

**Issue by pahdo** [sfbrigade/nltweets#25]
Noticed that new members without prior data science experience don't have great resources to ramp up and start doing interesting experiments. Since a lot of our work is exploratory, we benefit from as many members as possible doing experiments in parallel. Thinking about building a set of CodeLabs that cover-

0. Using the Twitter API https://github.com/sfbrigade/nltweets/issues/62
1. NLTK/Spacy for preprocessing text data https://github.com/sfbrigade/nltweets/issues/41
2. Classification with scikit-learn https://github.com/sfbrigade/nltweets/issues/42

Ideas:
3. Using Tweepy to scrape tweets
4. Scikit-learn for transforming text data into document embeddings
5. Interpreting the results of an experiment

Seems like a good investment to prioritize this now for greater velocity later.

**Comment by frhino** [comment]
This is exactly what this project is all about! Thanks Daniel!

**Comment by frhino** [comment]
@pahdo What do you think about breaking this issue down into separate issues of creating the following CodeLabs?
* Using Tweepy to scrape tweets
* Scikit-learn for transforming text data into document embeddings
* Interpreting the results of an experiment

**Comment by frhino** [comment]
@pahdo Awesome job on the three codelabs you created. Want to close this and open a new issue for any new codelab we want to create?

**Comment by pahdo** [comment]
Okay, for sure. Closing this issue.
